src/drivers/intel/dmic.c

```diff
@@ -1588,6 +1561,8 @@ static int dmic_probe(struct dai *dai)
 	if (dai_get_drvdata(dai))
 		return -EEXIST; /* already created */
 
+	/* TODO: I don't understand why next seem to need to be RUNTIME_SHARED */
```

Comment:
isn't it because it's allocated on core 0 and can be used by other cores?

Reply:
Oh, I forgot to remove the comment. I think I got it clear after understanding that probe and set_config happen on core 0 always.

Reply:
Done.